products/error_tracking/frontend/scenes/ErrorTrackingIssueScene/ScenePanel.tsx

```diff
@@ -65,13 +62,6 @@ const IssueModalContent = ({ issueId }: { issueId: string }): JSX.Element => {
                     eventLoading={initialEventLoading}
                     label={tagRenderer(selectedEvent)}
                 />
-                <Metadata>
-                    <EventsTable
-                        issueId={issueId}
-                        selectedEvent={selectedEvent}
-                        onEventSelect={(selectedEvent) => (selectedEvent ? selectEvent(selectedEvent) : null)}
-                    />
-                </Metadata>
```

Comment:
loading this part was hanging the browser, so lets remove it and keep it even leaner